CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Station Simulator Releases #
 
+### Version 0.3.8
+
+- Added support for Remote unlock connector
+- Added TxStartPoint and TxStopPoint configurations
```

Comment:
Could you please also add `Logging is moved from console to log file`?

Reply:
Done